hal/halx86/acpi/madt.c

```diff
@@ -12,77 +13,98 @@
 /* ACPI_BIOS_ERROR defined in acoutput.h and bugcodes.h */
 #undef ACPI_BIOS_ERROR
 #include <smp.h>
+
+// Note: It looks like DPRINT*() do nothing at the time this code usually runs, for some reason (= too early?).
```

Comment:
Anything that could be explained/improved about that?
They work fine when adding a test call at another time.

Reply:
They can also work with the FrLdr DbgPrint hack. Separate issue.